Content types configuration

mahajan-dsai opened the pull request:
To assure working version is synchronised following database errors

mahajan-dsai committed:
High Level Content-Types  for Events, Speakers, Courses, Job Listings, Team

mahajan-dsai committed:
Event Sectiobn Heading

Added Event Category - section heading in Event type

mahajan-dsai committed:
Course Fields

Fields - Course Length, Course Date, Course Venue, Course Image, Course Description - added

mahajan-dsai committed:
Changes to Events content type

Fields -  Date, Venue, Image, URL, Teaser - added; Category removed.

mahajan-dsai committed:
Speaker content type fields created

Speaker fields Photo, Name, Bio, and URLS for LinkedIn, Twitter, Website and YouTube, created.

mahajan-dsai committed:
Team Content Type fields created

Team fields - name, photo, bio, and links to LinkedIn, Twitters, Website, YouTube - created

mahajan-dsai committed:
Job Listings fields added

Fields per  https://jobs.associations.net.au/, Seek sites. Standard formats will need to be defined for some fields during data entry and automation so that search and listings are a seemless experience.

mahajan-dsai committed:
Course - field order and descriptions

Course content type order now: Image, Title, Description, Date, Course Length, Venue, Price
Added Price

oklmnk commented on config/default/field.field.node.event.field_event_teaser.yml:
```diff
@@ -0,0 +1,21 @@
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: text
```
Please convert the event teaser to full-text multiline editor and move the field below the title.

![Screenshot from 2019-12-05 21-31-37](https://user-images.githubusercontent.com/47801537/70227415-a8aad780-17a6-11ea-8446-f30bc3c8df52.png)

oklmnk commented on config/default/node.type.event.yml:
```diff
@@ -9,7 +9,7 @@ third_party_settings:
       - main
     parent: 'main:'
-name: Event
+name: 'Event Title'
```
Rename event title to 'Event'

mahajan-dsai committed:
Numerous changes to content types

changes across all content types - field types, field ordering

mahajan-dsai committed:
Changed fields to plain text

Content-types-configuration:: updated numerous fields to palin text per issue #8 on github

mahajan-dsai committed:
Views and field updates

Views for Courses, Events, Speakers, Team, Blogs; additional fields for event time, map ref, "follow"; menu ordering

mahajan-dsai committed:
Course time

Added optional field for course time

mahajan-dsai committed:
Added Blog - feed

feed with /blog/rss link and override display
